docs/examples/Power Flow Example.ipynb

```diff
@@ -348,6 +381,155 @@
+    "        ComponentType.line: [\"id\", \"p_from\"],  # line output columns id and p_from\n",
+    "    },\n",
+    ")\n",
+    "print(\"List of component types in result dataset\")\n",
+    "print(list(output_data))\n",
+    "print(\"List of attribute types in line result\")\n",
+    "print(list(output_data[ComponentType.line]))\n",
+    "print(\"------line result------\")\n",
+    "print(output_data[ComponentType.line])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e564346b",
+   "metadata": {},
+   "source": [
+    "You can also mix output types between components. In this example:\n",
+    "\n",
+    "* `None` requests row-based data\n",
+    "* a list of attribute names requests columns for those attributes (as before)\n",
+    "* `ComponentAttributeFilterOptions.ALL` requests columns for all supported attributes of a component."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "f52dbdd7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "List of component types in result dataset and associated data types\n",
+      "{'ComponentType.node': <class 'numpy.ndarray'>, 'ComponentType.line': <class 'dict'>, 'ComponentType.sym_load': <class 'dict'>}\n",
+      "------node result------\n",
+      "('id', 'energized', 'u_pu', 'u', 'u_angle', 'p', 'q')\n",
+      "------line result attributes------\n",
+      "['id', 'p_from']\n",
+      "------sym_load result attributes------\n",
+      "['id', 'energized', 'p', 'q', 'i', 's', 'pf']\n"
+     ]
+    }
+   ],
+   "source": [
+    "output_data = model.calculate_power_flow(\n",
+    "    symmetric=True,\n",
+    "    error_tolerance=1e-8,\n",
+    "    max_iterations=20,\n",
+    "    calculation_method=CalculationMethod.newton_raphson,\n",
+    "    output_component_types={\n",
+    "        ComponentType.node: None,  # node output as row-based\n",
+    "        ComponentType.line: [\"id\", \"p_from\"],  # line output columns id and p_from\n",
+    "        ComponentType.sym_load: ComponentAttributeFilterOptions.everything,  # all sym_load attributes as columns\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "print(\"List of component types in result dataset and associated data types\")\n",
+    "print({str(component_type): type(component_data) for component_type, component_data in output_data.items()})\n",
+    "print(\"------node result------\")\n",
+    "print(output_data[ComponentType.node].dtype.names)\n",
+    "print(\"------line result attributes------\")\n",
+    "print(list(output_data[ComponentType.line].keys()))\n",
+    "print(\"------sym_load result attributes------\")\n",
+    "print(list(output_data[ComponentType.sym_load].keys()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14a34c07",
+   "metadata": {},
+   "source": [
+    "Finally, it is possible to request columnar data for all attributes using the `ComponentAttributeFilterOptions.ALL` as a shorthand."
```

Comment:
Isn't this line sort of duplicated from above? Check the only other text line with `ComponentAttributeFilterOptions.ALL`

Reply:
I just realized... We have to change this to `*.everything`.